finality-aleph/src/network/manager/service.rs

```diff
@@ -307,21 +287,16 @@ where
         pre_session: PreValidatorSession,
         result_for_user: Option<oneshot::Sender<mpsc::UnboundedReceiver<D>>>,
     ) -> Result<ServiceActions<M, NI::AddressingInformation>, SessionHandlerError> {
-        match self.update_validator_session(pre_session.clone()).await {
-            Ok((actions, data_from_network)) => {
+        self.update_validator_session(pre_session.clone())
```

Comment:
You no longer need to `clone` here.

Btw, we really should get clippy back in the pipeline. xd